ceed-basis.c

```diff
@@ -407,12 +407,12 @@ int CeedBasisGetColocatedGrad(CeedBasis basis, CeedScalar *colograd1d) {
 ///     \ref CEED_EVAL_GRAD to obtain gradients.
 /// @param u input vector
 /// @param v output vector
-int CeedBasisApply(CeedBasis basis, CeedTransposeMode tmode, CeedEvalMode emode,
-                   const CeedScalar *u, CeedScalar *v) {
+int CeedBasisApply(CeedBasis basis, CeedInt nelem, CeedTransposeMode tmode,
```

Comment:
The man page above needs to be updated for the new parameter and implied input and output array ordering.

Reply:
Good point about the manpages. As for the implied ordering, I am a bit unsure. It didn't sound like the GPU backends would like to interlace elements. Would it be fair to state that the `RestrictionCreateBlocked` for a given backend will enforce the ordering required for the `BasisApply`?